libs/src/index.js

```diff
@@ -174,7 +174,8 @@ class AudiusLibs {
     comstockConfig,
     captchaConfig,
     isServer,
-    isDebug = false
+    isDebug = false,
+    enableUserReplicaSetManagerContract = false
```

Comment:
I think this should be flipped and it should be true by default - this allows mad dog in CI to continue giving us coverage

Reply:
or we need to special case CI

Reply:
ok if i just modify mad-dog libs init instead?

Reply:
doing, one sec

Reply:
yeah modifying this in mad dog would be ideal. don't want to enable this by accident

Reply:
fixed